SwrveSDK/src/main/java/com/swrve/sdk/locationcampaigns/model/LocationCampaign.java

```diff
@@ -9,14 +9,14 @@
 
 public class LocationCampaign  {
 
-    private final int id;
+    private String id;
```

Comment:
Its an int currently. It could be a UUID if we wanted to. The sdk can handle either. The LocationMessage id is a string also.